### ISSUE TITLE
Query Parameters don't work when passing variables to nested fields

### ISSUE THREAD

**Issue by peternorbeck** [neo4j-graphql/neo4j-graphql-js#20]
$year variable works fine but the $first variable fails.

```
query ($year: Int = 2016, $first: Int = 2) {

  Movie(year: $year) {
    title
    year
    similar(first: $first) {
       title
    }
  }
}
```

Error:
```
{
  "errors": [
    {
      "message": "Variable `undefined` not defined (line 1, column 192 (offset: 191))\n\"MATCH (movie:Movie {year:2016}) RETURN movie { .title , .year ,similar: [ x IN apoc.cypher.runFirstColumn(\"MATCH (this)--(:Genre)--(o:Movie) RETURN o\", {this: movie}, true) | x { .title }][..undefined] } AS movie SKIP 0\"\n                                                                                                                                                                                                ^",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "Movie"
      ]
    }
  ],
  "data": {
    "Movie": null
  }
}
```

**Comment by peternorbeck** [comment]
I updated to the latest versions of all grapql dependencies and it started to work so this issue can be closed.

**Comment by mayurimhatre** [comment]
@peternorbeck what version are you using for below dependencies?
graphql
graphql-tools
neo4j-driver
neo4j-graphql-js

**Comment by neurofoo** [comment]
Having the same issue in both client side js and igraphql 

On server, running:
"neo4j-driver": "^1.5.3",
"neo4j-graphql-js": "^0.1.8"

**Comment by johnymontana** [comment]
Thanks @peternorbeck for reporting and @neurofoo @mayurimhatre for confirming. This should now be fixed in v0.1.12. Please test and report back if the problem persists.
